src/api/hooks/razorpay.ts

```diff
@@ -23,9 +23,7 @@ export default async (req, res) => {
         res.sendStatus(400);
         return;
       }
-      data = _.isObject(req.body)
-        ? req.body?.data?.toString()
```

Comment:
@SGFGOV any particular reason for you to use `req.body.data` to access the data.

Or was this due to backward compatibility.